models/user.go

```diff
@@ -388,16 +388,16 @@ func (u *User) NewGitSig() *git.Signature {
 	}
 }
 
-// EncodePasswd encodes password to safe format.
-func (u *User) EncodePasswd() {
+// HashPassword hashes a password using PBKDF.
+func (u *User) HashPassword() {
 	newPasswd := pbkdf2.Key([]byte(u.Passwd), []byte(u.Salt), 10000, 50, sha256.New)
 	u.Passwd = fmt.Sprintf("%x", newPasswd)
 }
```

Comment:
I'll just run `user.HashPassword()` twice to make my password twice as secure :trollface: (I'll send a PR after this is merged 😂 )

Reply:
That can never be as secure as running your password through `<?php md5(md5($password));`! Haha! I will never be hacked this way!